patches/openwrt/0007-xrx200-ensure-the-correct-version-for-fritz7360.patch

```diff
@@ -0,0 +1,94 @@
+From fa3af8c6583b5338499f10aec926f49d399e84ee Mon Sep 17 00:00:00 2001
+From: Grische <github@grische.xyz>
+Date: Sun, 18 Sep 2022 14:03:16 +0200
+Subject: [PATCH] xrx200: ensure the correct version for fritz7360
+
+---
+ .../base-files/lib/preinit/01_sysinfo.sh      | 75 +++++++++++++++++++
+ 1 file changed, 75 insertions(+)
+ create mode 100644 target/linux/lantiq/xrx200/base-files/lib/preinit/01_sysinfo.sh
+
+diff --git a/target/linux/lantiq/xrx200/base-files/lib/preinit/01_sysinfo.sh b/target/linux/lantiq/xrx200/base-files/lib/preinit/01_sysinfo.sh
+new file mode 100644
+index 0000000000..f0520502ce
+--- /dev/null
++++ b/target/linux/lantiq/xrx200/base-files/lib/preinit/01_sysinfo.sh
+@@ -0,0 +1,75 @@
++set_sysinfo_xrx200() {
++    local board_name=$1
++    local model=$2
++
++    echo "${board_name}" > /tmp/sysinfo/board_name
++    echo "${model}" > /tmp/sysinfo/model
++}
++
++do_sysinfo_xrx200_for_fritz7360_model() {
++    local board_name=$1
++    local model=$2
++    local urlader_version urlader_memsize urlader_flashsize
++    local hexdump_format='4/1 "%02x""\n"'
++
++    # Values are based on urlader-parser-py
++    # https://github.com/grische/urlader-parser-py/blob/42970bf8dec7962317df4ff734c57ebf36df8905/parser.py#L77-L84
++    urlader_version="$(dd if=/dev/mtd0ro bs=1 skip=$((0x580+0x0)) count=4 | hexdump -e "${hexdump_format}")"
++    if [ "${urlader_version}" != "00000003" ]; then
++        >&2 echo "Warning: unexpected urlader version found: ${urlader_version}"
```

Comment:
Use the OpenWrt logging framework: https://openwrt.org/docs/guide-user/base-system/log.essentials

Reply:
Thanks, I did not know this one. Should be fixed now.